include/exadg/operators/inverse_mass_operator.h

```diff
@@ -48,6 +50,30 @@ struct InverseMassOperatorData
   {
   }
 
+  // Get optimal in the sense of (most likely) fastest implementation type of the inverse mass
+  // operator depending on the approximation space.
+  template<int dim>
+  static InverseMassType
+  get_optimal_inverse_mass_type(dealii::FiniteElement<dim> const & fe,
+                                ElementType const                  element_type)
+  {
+    if(fe.conforms(dealii::FiniteElementData<dim>::L2))
+    {
+      if(element_type == ElementType::Hypercube)
+      {
+        return InverseMassType::MatrixfreeOperator;
+      }
+      else
+      {
+        return InverseMassType::ElementwiseKrylovSolver;
```

Comment:
If we work more frequently with simplex meshes, we should do a different approach - elementwise Krylov is likely a good option for deformed mesh elements, but affine meshes, which are the majority of meshes in practice, one should use the inverted mass matrix in reference coordinates and scaling by the Jacobian determinant. Of course, such a strategy is not the purpose of this PR and the alternative option is not available in ExaDG right now, but I thought it is good to mention it.